purchases/src/test/java/com/revenuecat/purchases/PurchasesTest.kt

```diff
@@ -143,43 +116,7 @@ class PurchasesTest {
 
     private val mockLifecycle = mockk<Lifecycle>()
     private val mockLifecycleOwner = mockk<LifecycleOwner>()
```

Comment:
There might be some properties that will probably be needed by different test classes and could be moved to `BasePurchasesTest`, but we can extract them whenever we need to.